cla_public/templates/privacy.html

```diff
@@ -318,7 +323,13 @@ <h2>{{ _('How we use cookies') }}</h2>
     </li>
   </ul>
   <p>
-    {% trans cookie_url = url_for('.cookies') %}You can read more about <a href="{{ cookie_url }}">how cookies are used</a>.{% endtrans %}
+    {% trans %}The online service also uses cookies from the Government Digital Service (GDS). These cookies collect information to help improve the user experience across government digital services.{% endtrans %}
+    <p>
+    {% trans %}You can change <a href="https://checklegalaid.service.gov.uk/cookie-settings">your cookie settings</a> to opt in or out of how different cookies are used.{% endtrans %}
```

Comment:
You should use `url_for('.cookie_settings')` instead of the full link here, the same way you did it for cookies page link

Reply:
Thanks @said-moj - I've changed using your code snippet, and looks good on staging. Can you review the PR again?